preprocessing/preprocessingImages/cornerDetection.py

```diff
@@ -0,0 +1,56 @@
+import math
+import numpy as np
+import cv2 as cv
+
+# helper for filtering points
+def distance(pt1, pt2):
+        (x1, y1), (x2, y2) = pt1, pt2
+        dist = math.sqrt( (x2 - x1)**2 + (y2 - y1)**2 )
+        return dist
+
+
+def get_contours(filename):
+    img = cv.imread(filename)
+
+    # convert the input image into grayscale color space 
+    grayImage = cv.cvtColor(img, cv.COLOR_BGR2GRAY) 
+
+    # modify the data type setting to 32-bit floating point and get corners 
+    operatedImage = np.float32(grayImage) 
+    dst = cv.cornerHarris(operatedImage, 2, 5, 0.07)
```

Comment:
It might help to give parameters/numbers like this a good variable name, so that others looking at the file can see what they are meant to be. Keeping them at the top of the function, or as parameters with default values could also be useful for readability and changeability